ajax/libs/ScrollTrigger/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ScrollTrigger",
+  "version": "0.3.6",
+  "filename": "ScrollTrigger.min.js",
+  "description": "Triggers classes on html elements based on the scroll position. Uses requestAnimationFrame so it doesn't jack the users scroll.",
```

Comment:
Is there any reason why it's different from the upstream?
Ref: https://github.com/terwanerik/ScrollTrigger/blob/master/package.json#L4